web/modules/custom/react_tools/tools/config/webpack.config.prod.js

```diff
@@ -0,0 +1,176 @@
+'use strict';
+
+const autoprefixer = require('autoprefixer');
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const ManifestPlugin = require('webpack-manifest-plugin');
+const InterpolateHtmlPlugin = require('react-dev-utils/InterpolateHtmlPlugin');
+const eslintFormatter = require('react-dev-utils/eslintFormatter');
+const paths = require('./paths');
+const getClientEnvironment = require('./env');
+const apps = require('./apps');
+
+const publicPath = paths.servedPath;
+const shouldUseRelativeAssetPaths = publicPath === './';
+const shouldUseSourceMap = false;
+const publicUrl = publicPath.slice(0, -1);
+const env = getClientEnvironment(publicUrl);
+
+if (env.stringified['process.env'].NODE_ENV !== '"production"') {
+  throw new Error('Production builds must have NODE_ENV=production.');
+}
+
+const cssFilename = 'static/css/[name].min.css';
+const extractTextPluginOptions = shouldUseRelativeAssetPaths
+  ? // Making sure that the publicPath goes back to to build folder.
+    { publicPath: Array(cssFilename.split('/').length).join('../') }
+  : {};
+
+module.exports = {
+  bail: true,
+  devtool: shouldUseSourceMap ? 'source-map' : false,
+  entry: apps.entryPoints,
+  output: {
+    path: paths.appBuild,
+    filename: 'static/js/[name].min.js',
+    chunkFilename: 'static/js/[name].chunk.js',
+    publicPath: publicPath
+  },
+  resolve: {
+    modules: ['node_modules'],
+    extensions: ['.web.js', '.mjs', '.js', '.json', '.web.jsx', '.jsx'],
+    alias: {
+      'react-native': 'react-native-web',
+    }
+  },
+  module: {
+    strictExportPresence: true,
+    rules: [
+      // TODO: Disable require.ensure as it's not a standard language feature.
```

Comment:
is this TODO already solved? I don't see any require.ensure

Reply:
This was left in from create-react-app itself. Since we are ejected we can remove this. I am still working on cleanup tasks which will be in the next commit.